test/Tests/src/Data/Time/Date_Time_Spec.enso

```diff
@@ -783,22 +783,22 @@ spec_with name create_new_datetime parse_datetime nanoseconds_loss_in_precision=
             t1.date_diff t2 Time_Period.Day . should_equal 32
 
             t1.date_diff t2 Time_Period.Hour . should_equal 770
-            t1.date_diff (Date_Time.new 2021 11 3 12 15 0) Time_Period.Hour . should_equal 2
+            t1.date_diff (Date_Time.new 2021 11 3 12 15 0 zone=Time_Zone.utc) Time_Period.Hour . should_equal 2
```

Comment:
This is deliberate.

If we use `create_new_datetime` everywhere, we test how values from one language tested at given time interact with each other. By using `Date_Time.new` for the argument **and** still using `create_new_datetime` for the receiver (`self = t1`), we test how an Enso value interoperates with a value coming from another language - i.e. we check mixing of values from different languages with each other - which also _needs_ to be supported as the values should all be interchangeable regardless of what language they come from.

I don't remember if this was added because there bugs with that or if it was just to ensure we test a wider range of scenarios, but surely it was done deliberately to test more interesting 'mixing' cases.